model/clusters_mgmt/v1/cluster_registration_type.model

```diff
@@ -15,30 +15,38 @@ limitations under the License.
 */
 
 // Registration of a new cluster to the service.
+//
+// For example, to register a cluster that has been provisioned outside
+// of this service, send a a request like this:
+//
+// ```http
+// POST /api/clusters_mgmt/v1/register_cluster HTTP/1.1
+// ```
+//
+// With a request body like this:
+//
+// ```json
+// {
+//   "external_id": "d656aecf-11a6-4782-ad86-8f72638449ba",
+//   "subscription_id": "...",
+//   "organization_id": "..."
+// }
+// ```
 struct ClusterRegistration {
 	// Subscription identifier of the cluster generated by the account
 	// manager.
 	SubscriptionID String
 
-	// Optional Organization identifier of the cluster generated by the
+	// Organization identifier of the cluster generated by the
 	// account manager.
 	OrganizationID String
 
 	// Identifier of the cluster generated by the installer.
-	//
-	// For example, to register a cluster that has been provisioned outside
-	// of this service, send a a request like this:
-	//
-	// ```http
-	// POST /api/clusters_mgmt/v1/register_cluster HTTP/1.1
-	// ```
-	//
-	// With a request body like this:
-	//
-	// ```json
-	// {
-	//   "external_id": "d656aecf-11a6-4782-ad86-8f72638449ba"
-	// }
-	// ```
 	ExternalID String
+
+	// Optional Display Name for the cluster.
+	DisplayName String
```

Comment:
Just curious, what is the status with DisplayName, I was under the impression we had removed DisplayName from CS or has that change been reverted?

Reply:
it was reverted

Reply:
Ah cool cool cool

Reply:
@tbrisker @ciaranRoche  we dont want to use display name it was reverted in CS until AMS fixes their logic. however we want new users to stop using the display name . can you please remove it